app/views/compliance_hosts/show.html.erb

```diff
@@ -8,7 +8,8 @@
     <% if data.has_data? %>
     <div class="col-md-4">
       <div class="stats-well">
-        <h4 class="header ca"><%= _('%s latest report') % policy.name %></h4>
+        <h4 class="header ca"><%= display_link_if_authorized(_('%s latest report') % policy.name,
```

Comment:
it would be better to use `link_to_if_authorized` that would keep the text but it would not be clickable

see the difference
![0label](https://user-images.githubusercontent.com/109773/32735307-a77a6ff0-c894-11e7-8a13-f8bab2b6daf0.png)
![1label](https://user-images.githubusercontent.com/109773/32735308-a9d96b70-c894-11e7-95b6-a8bf80647152.png)